pandas/tests/frame/test_analytics.py

```diff
@@ -1201,6 +1201,13 @@ def test_isin_df(self):
         expected['B'] = False
         tm.assert_frame_equal(result, expected)
 
+    def test_isin_tuples(self):
+        df = pd.DataFrame({'A': [1, 2, 3], 'B': ['a', 'b', 'f']})
```

Comment:
Can you add the original issue number as a comment on the first line ? 
just like `# GH16394`